src/rgx/block.rgx

```diff
@@ -0,0 +1,59 @@
+([^A-Za-z0-9-_./+%25]|\/\.\.|%25[^2-7a-fA-F][^0-9a-fA-F]|%25(2[26]|3([aA]|[dD])|5[cC]|7[cC])|//etc/|
```

Comment:
It would significantly improve readability and maintainability if the regex would be split one ORed expression per line and sorted alphabetically.